html/hobbies.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Kimberley Zell: Hobbies</title>
+    <link href="../styles/style.css" rel="stylesheet">
+  </head>
+  <body>
+    <header>
+
+      <h1>Kimberley Zell</h1>
+
+      <nav>
+
+        <ul>
+          <li>
+            <a href="../html/index.html">Home</a>
+          </li>
+          <li>
+            <a href="../html/portfolio.html">Portfolio</a>
+          </li>
+          <li>
+            <a href="../html/hobbies.html">Hobbies</a>
+          </li>
+          <li>
+            <a href="../html/about.html">About</a>
+          </li>
+          <li>
+            <a href="https://gist.github.com/kimpossible1/595794b33272b668d8f760be8a1aff3c"target="_blank"> Resume </a>
+          </li>
+        </ul>
+      </nav>
+
+    </header>
+
+    <section class="hobby-banner" class="wrapper">
+      <h1 id="hobby-headline">
+        LIVE --> EXPLORE --> YOGA EVERYWHERE
+      </h1>
+  </section>
+
+  <section class="image-collage" class="wrapper">
+
+    <img src="/Users/kzell/Documents/Ada/Static-Site/images/Indo-water-backbend.jpg">
+
+    <img src="/Users/kzell/Documents/Ada/Static-Site/images/Thailand-lizard-balance.jpg">
```

Comment:
These are all absolute paths - they go all the way from the root of your filesystem (the directory `/`) to where the file lives. While this works on your computer, it does not work on mine, since I don't have a directory `/Users/kzell`. Instead you should use relative paths, like `../images/Thailand-island-view.jpg`.

A quick search-and-replace of `/Users/kzell/Documents/Ada/Static-Site/` -> `../` seems to fix the problem.